server/routes/api/businesses.js

```diff
@@ -29,14 +27,15 @@ router.get("/getThree", (req, res) => {
 });
 
 
-router.post("/byCategory", (req, res) => {
+router.get("/getByCategory/:category", (req, res) => {
   Business.findAll({
-    where: {
-      type: req.body.category,
-    },
-  }).then((Businesses) => res.json(Businesses))
+    where: { category: req.params.category },
+  })
+    .then((Businesses) => res.json(Businesses))
```

Comment:
you don't have to change this now. But Idk if res.json() will throw an error if sequelize returns null? It might be safer to use `res.status(200).send({ Businesses })`